src/connectors/textToSpeech/textToSpeechConnector.ts

```diff
@@ -1,8 +1,8 @@
 import * as tempy from 'tempy';
 import * as fs from 'fs';
+import * as TextToSpeechV1 from 'ibm-watson/text-to-speech/v1';
```

Comment:
Why tho ?

Reply:
Dann ist es einheitlich (überall `import` statt `require`), und `import` ist ein ES6 Standard, während es `require` nur in Node gibt